x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_field/description.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiDescribedFormGroup, EuiFormRow } from '@elastic/eui';
+
+export const Description: FC = memo(({ children }) => {
+  const title = i18n.translate('xpack.ml.newJob.wizard.pickFieldsStep.splitRareField.title', {
+    defaultMessage: 'Rare field',
+  });
+  return (
+    <EuiDescribedFormGroup
+      title={<h3>{title}</h3>}
+      description={
+        <FormattedMessage
+          id="xpack.ml.newJob.wizard.pickFieldsStep.splitField.description"
+          defaultMessage="Select a field to partition analysis by. Each value of this field will be modeled independently individually."
```

Comment:
@szabosteve i missed this when in your last commit. This text needs updating to describe the rare field.
Currently it's a copy of the split field text

Reply:
@jgowdyelastic Thanks for the ping, please see my suggestion below.
```suggestion
          defaultMessage="Select a field in which to detect rare values."
```